### PR TITLE
New 2022 officehours

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: uw-research-computing/get-help.md [hunk 1]
```diff
@@ -42,14 +42,13 @@ expect a first response within a few business hours.
...
-> ## Office Hours are cancelled on Tuesday, August 24
--->
+> ## Office Hours are cancelled the week of December 27 - 31
 
 In light of [measures being taken by campus regarding COVID-19](https://covid19.wisc.edu/), 
 we have replaced in-person office hours with virtual office hours, office hours are:
 
--   **Tuesday/Thursday afternoons, from 3:00 - 4:45 pm.** 
+-   **Tuesday morning: 10:30am - 12:00pm.** 
+-   **Thursday afternoon: 3:00 - 4:45 pm.**
```
Comment (ChristinaLK) [inline uw-research-computing/get-help.md:51]:
```suggestion
-   **Thursday afternoon: 3:00 - 4:30 pm.** 
```

File: _includes/footer.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
         <div class="uw-footer-contact">
             <h3 class="uw-footer-header">Contact Us</h3>
             <ul class="uw-contact-list">
-                <li class="uw-contact-item">Office Hours: <a href="{{ '/uw-research-computing/get-help.html' | relative_url }}">Tues/Thurs, 3-4:45pm</a></li>
+                <li class="uw-contact-item">Office Hours: <a href="{{ '/uw-research-computing/get-help.html' | relative_url }}">Tues: 10:30-12; Thurs: 3-4:30</a></li>
```
Comment (rachellombardi) [inline _includes/footer.html:11]:
```suggestion
                <li class="uw-contact-item">Office Hours: <a href="{{ '/uw-research-computing/get-help.html' | relative_url }}">Tues: 10:30am-12pm; Thurs: 3-4:30pm</a></li>
```

**Review by rachellombardi — COMMENTED** [review]
Looks good!

**Review by CannonLock — APPROVED** [review]
LGTM.

Screen Shot 2021-12-23 at 1 25 48 PM (3)

**Pull request merged**